### ISSUE TITLE
The dynamic method calls in form's entity for underscore values

### ISSUE THREAD

**Issue by sszdh** [phalcon/cphalcon#10561]
Hi my friends,

I think phalcon should camelize every values before make a method name then makes call.

For example consider this: I have a `foo_bar` field in my model and  model passed as an entity into the form:

```php
class FooClass extends \Phalcon\Mvc\Model
{
	protected foo_bar;
	
	public function setFooBar() {}
	
	public function getFooBar() 
	{
		return \App\Utils::makeSomeModifications($this->foo_bar)
	}
}

class MyForm extends \Phalcon\Forms\Form
{
	public function initialize(FooClass $foo = null)
    {
        $v1 = $this->getValue('foo_bar'); // <-- This calls FooClass::getFooBar() returns correct value
        $v2 = $course->foo_bar;	// <-- This tries to call FooClass::getfoo_bar 
	}
}
```

This is because the [`__get()`](https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model.zep#L4095) magic method in model uses `Text::camelize` but the [`getValue()`](https://github.com/phalcon/cphalcon/blob/master/phalcon/forms/form.zep#L586) method of the form base class doesn't!

Actually I think everywhere in the code (not only model and forms) , when we'r trying to make dynamic method call `(get . "blahblah", set . "blahblah", etc)` , we should `camelize` method name to 
save integrity and standards.

Thanks

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch

**Comment by sszdh** [comment]
Great.
